packages/discount-code-exporter/test/main.spec.js

```diff
@@ -1,11 +1,220 @@
-import findSum from '../src/main'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import streamtest from 'streamtest'
+import DiscountCodeExport from '../src/main'
 
-describe('findSum', () => {
-  it('should be a function', () => {
-    expect(typeof findSum).toBe('function')
+describe('DiscountCodeExport', () => {
+  const logger = {
+    error: () => {},
+    warn: () => {},
+    info: () => {},
+    verbose: () => {},
+  }
+
+  let codeExport
+  beforeEach(() => {
+    codeExport = new DiscountCodeExport({
+      apiConfig: {
+        projectKey: 'asafaelhn',
```

Comment:
I'd rather see a meaningful `projectKey` than junk even if it doesn't make any actual request

`projectKey: 'random-project-key'`